cluster/providers/ovirt/setup.sh

```diff
@@ -2,14 +2,14 @@
 
 set -ex
 
-kubectl apply -f ovirt/patch.yml
-kubectl apply -f ovirt/fakeovirt_deployment.yml
-kubectl apply -f ovirt/ovirt_imageio_deployment.yml
+kubectl apply -f cluster/providers/ovirt/patch.yml
```

Comment:
do we still need to do that? (and if not, can we drop patch.yml?)

Reply:
yeap , it can be dropped.